specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview/examples/OpenShiftManagedClustersCreate_Update.json

```diff
@@ -0,0 +1,214 @@
+{
+  "parameters": {
+    "api-version": "2019-10-27-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "parameters": {
+      "location": "location1",
+      "tags": {
+        "tier": "production",
+        "archv2": ""
+      },
+      "properties": {
+        "openShiftVersion": "v3.11",
+        "networkProfile": {
+          "vnetCidr": "10.0.0.0/8"
+        },
+        "masterPoolProfile": {
+          "count": 3,
+          "vmSize": "Standard_D4s_v3",
+          "subnetCidr": "10.0.0.0/24",
+          "apiProperties": {
+            "privateAPIServer": true
+          }
+        },
+        "agentPoolProfiles": [
+          {
+            "name": "infra",
+            "role": "infra",
+            "count": 2,
+            "vmSize": "Standard_D4s_v3",
+            "osType": "Linux",
+            "subnetCidr": "10.0.0.0/24"
+          },
+          {
+            "name": "compute",
+            "role": "compute",
+            "count": 4,
+            "vmSize": "Standard_D4s_v3",
+            "osType": "Linux",
+            "subnetCidr": "10.0.0.0/24"
+          }
+        ],
+        "routerProfiles": [
+          {
+            "name": "default"
+          }
+        ],
+        "authProfile": {
+          "identityProviders": [
+            {
+              "name": "Azure AD",
+              "provider": {
+                "kind": "AADIdentityProvider",
+                "clientId": "clientId",
+                "secret": "secret",
+                "tenantId": "tenantId",
+                "customerAdminGroupId": "customerAdminGroupId"
+              }
+            }
+          ]
+        },
+        "monitorProfile": {
+          "enabled": true,
+          "workspaceResourceID": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.OperationalInsights/workspaces/workspacename1"
+        },
+        "refreshCluster": true
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/openShiftManagedClusters/clustername1",
+        "location": "location1",
+        "name": "clustername1",
+        "tags": {
+          "archv2": "",
+          "tier": "production"
+        },
+        "type": "Microsoft.ContainerService/OpenShiftManagedClusters",
+        "properties": {
+          "provisioningState": "Updating",
+          "openShiftVersion": "v3.11",
+          "clusterVersion": "osa.1",
+          "fqdn": "df9337748a4b.location1.cloudapp.azure.com",
+          "publicHostname": "openshift.1225bdf9337748a4bd3a.location1.azmosa.io",
+          "networkProfile": {
+            "vnetCidr": "10.0.0.0/8"
```

Comment:
Same, `managementSubnetCIDR` is returned

Reply:
let's keep this file as "public" cluster example. 
I've added a new file with example for the private cluster: `specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview/examples/OpenShiftManagedClustersPrivateClusterCreate_Update.json`